src/explore-education-statistics-admin/src/components/editable/__tests__/ReorderableAccordion.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import noop from 'lodash/noop';
+import Button from '@common/components/Button';
+import ReorderableAccordion from '../ReorderableAccordion';
+
+describe('ReorderableAccordion', () => {
+  test("doesn't show controls if `canReorder` is false", () => {
+    const canReorder = false;
+
+    render(
+      <ReorderableAccordion
+        canReorder={canReorder}
+        id="test"
+        onReorder={noop}
+        heading="test"
+        onSectionOpen={noop}
+        onToggleAll={noop}
+      >
+        {canReorder && <Button>Reorder</Button>}
+      </ReorderableAccordion>,
+    );
+
+    expect(
+      screen.queryByRole('button', { name: 'Reorder' }),
+    ).not.toBeInTheDocument();
+  });
+
+  test('shows controls if `canReorder` is true', () => {
+    const canReorder = true;
+    render(
+      <ReorderableAccordion
+        canReorder={canReorder}
+        id="test"
+        onReorder={noop}
+        heading="test"
+        showOpenAll
+        onSectionOpen={noop}
+        onToggleAll={noop}
+      >
+        {canReorder && <Button>Reorder</Button>}
+      </ReorderableAccordion>,
+    );
+
+    expect(
+      screen.queryByRole('button', { name: 'Reorder' }),
+    ).toBeInTheDocument();
```

Comment:
This doesn't actually test that the 'Reorder sections' buttons exists. All we're testing is that the button in the children renders.